lib/core/generate.ts

```diff
@@ -36,6 +37,8 @@ export const generate = async (
     }. Generating type definitions...`
   );
 
+  removeUnusedTypeFiles(files);
```

Comment:
Since this is not an opt-in feature but required, I would consider this a breaking change.

I think we can either:

- Add a configuration option that makes this opt-in;
- Or, push an empty commit/re-write the history and force push with a commit denoting this as a breaking change. I believe for this, the commit body needs to contain `BREAKING CHANGE` (see [`semantic-release`](https://github.com/semantic-release/semantic-release#commit-message-format))